shared/components/pages/take_action/take_action.component.js

```diff
@@ -1,44 +1,39 @@
-/* global module*/
+/* global module window*/
 
```

Comment:
I think in eslint config you can provide 

```
{
  "env": {"browser": true, "node": true}
}
```
 to avoid declaring globals at top.